Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -177,6 +177,8 @@ NetworkSession::NetworkSession(NetworkProcess& networkProcess, const NetworkSess
     setTrackingPreventionEnabled(parameters.resourceLoadStatisticsParameters.enabled);
 #endif
 
+    setBlobRegistryTopOriginPartitioningEnabled(parameters.blobRegistryTopOriginPartitioningEnabled);
```

Comment:
This needs a 'is' prefix, per WebKit coding style (booleans need a prefix):
`parameters.isBlobRegistryTopOriginPartitioningEnabled `